augur/generate.py

```diff
@@ -0,0 +1,266 @@
+#
+# Data generation module
+#
+
+import firecrown
+import sacc
+import copy
+import numpy as np
+
+
+def generate(config):
+    """ Generates data using dictionary based config.
```

Comment:
```suggestion
    """Generates data using dictionary based config.
```

I don't like the extra space here. I tend to follow the numpy docstring standards out of habit: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard